finite_state_sdk/__init__.py

```diff
@@ -377,6 +380,8 @@ def create_new_asset_version_and_upload_binary(token, organization_context, busi
             Description of the artifact. If not provided, the default is "Firmware Binary".
         quick_scan (bool, optional):
             If True, will upload the file for quick scan. Defaults to False (Full Scan). For details about Quick Scan vs Full Scan, please see the API documentation.
+        upload_method (str, optional):
+            Upload method to use. Defaults to "API".
```

Comment:
This documentation is much less descriptive than anywhere else specifically it doesn't include valid values.